.github/workflows/test.yml

```diff
@@ -66,5 +66,27 @@ jobs:
         go-version: '1.21'
     - name: Checkout code
       uses: actions/checkout@v3
-    - name: Test
-      run: make test-community
+    - name: Set up gotestsum
+      run: |
+        go install gotest.tools/gotestsum@latest
+        mkdir -p tmp/test-results
+    - name: Test 
+      run: |
+        CGO_ENABLED=1 gotestsum --junitfile tmp/test-results/test.xml --raw-command -- go test -json -tags=sources $(go list ./... | grep -v 'pkg/sources')
```

Comment:
Previously (and the make target) had `grep -v /vendor/ | grep -v pkg/detectors | grep -v pkg/sources)`. Why just the one exclusion now?

Reply:
Fixed; I had gotten the test make entry confused.